configure.ac

```diff
@@ -85,8 +85,15 @@ AM_CONDITIONAL(CXX14_ENABLED,test x$HAVE_CXX14 = x1)
 
 AM_CONDITIONAL(CXX11_ENABLED,test x$HAVE_CXX11 = x1)
 
+dnl Get CXXFLAGS and optional flags for this compiler
 ACSM_SET_CXX_FLAGS
 
+dnl Make sure it's safe to turn on FP exceptions
+ACSM_SET_FPE_SAFETY_FLAGS
+
+dnl provide the resulting flags to our Makefiles
+AC_SUBST(ACSM_CXXFLAGS_OPT)
```

Comment:
can we name this something other than `OPT`?

Reply:
It's literally the same flags we use in other contexts for `METHOD=opt`, which I thought was a decent choice for the tiny non-header-only bits of MetaPhysicL.

Less defensible is the fact that we only compile MetaPhysicL tests with a single set of compiler flags, rather than using multiple methods as with libMesh/TIMPI.  I probably should fix that, but "better test coverage" seemed way less urgent than "we can't even pass tests with recent clang versions".  And the specific tests we couldn't pass only fail at -O2 (it's the optimization that's generating code which triggers FPEs) so keeping our tests at that level seemed like the right place to start.

Reply:
ah alright that makes sense